src/starcheck.pl

```diff
@@ -136,6 +138,23 @@
 my $dither_file= get_file("$par{dir}/History/DITHER.txt*",'dither'); 
 my $radmon_file= get_file("$par{dir}/History/RADMON.txt*", 'radmon');
 
+# Check for characteristics.  Ignore the get_file required vs not API and just pre-check
+# to see if there is characteristics
+my $char_file;
+my $char_glob = "$par{dir}/${sosa_dir_slash}mps/ode/characteristics/CHARACTERIS*";
```

Comment:
I saw the thread indicated Brent thought it wasn't necessary to repeat it,
but I saw no resolution (was waiting for the mentioned followup to
mpweekly).

In src/starcheck.pl
<https://github.com/sot/starcheck/pull/122#discussion_r44535031>:

> @@ -136,6 +138,23 @@
>  my $dither_file= get_file("$par{dir}/History/DITHER.txt*",'dither');
>  my $radmon_file= get_file("$par{dir}/History/RADMON.txt*", 'radmon');
>
> +# Check for characteristics.  Ignore the get_file required vs not API and just pre-check
> +# to see if there is characteristics
> +my $char_file;
> +my $char_glob = "$par{dir}/${sosa_dir_slash}mps/ode/characteristics/CHARACTERIS*";

Brent indicated that this was not going to be in vehicle/ so I don't think
the sosa dir bit should be there.

—
Reply to this email directly or view it on GitHub
<https://github.com/sot/starcheck/pull/122/files#r44535031>.